### ISSUE TITLE
Feature request: allow specification of errorReportFile as a global option

### ISSUE THREAD

**Issue by jmb01** [OHDSI/DatabaseConnector#170]
Currently, the location of error report files is either user-specified within a given function call or, by default, taken to be the working directory. 

For cases when the user wants to store an errorReportFile outside of the working directory, but doesn't want to have to specify the path in every function call, it would be helpful to store it as option.

**Comment by anthonysena** [comment]
I'd also find this option useful - specifically when handling operations through Strategus, it would be nice to keep the SQL errors related to a specific folder used when executing a study. I can work on an implementation but wanted to ask @schuemie would you prefer this as an option that is set using `options()` or using `Sys.getenv()`?

**Comment by schuemie** [comment]
Good idea! I think an option would be better than an environmental variable because:
- This would be consistent with other options in DatabaseConnector like `"sqlRenderTempEmulationSchema"`
- This is likely something you want to set differently for different projects, so makes less sense as a system-wide variable.
